README.md

```diff
@@ -52,6 +52,7 @@ lua-posix 	| lua-posix             | 			| *1*
 python-devel	| python-dev		| >= 2.7		|
 python-cffi	| python-cffi		| >= 1.1		|
 python-six	| python-six		| >= 1.9		|
+python-yaml	| python-yaml		|			|
```

Comment:
Based on the configure check, I think `>= 3.10.0` should be added to the third column.